mkdocs.yml

```diff
@@ -54,3 +55,8 @@ markdown_extensions:
   - pymdownx.tasklist:
       custom_checkbox: true
   - pymdownx.tilde
+
+# Analytics tracking with GoatCounter
```

Comment:
What is this block for?

Reply:
to specify overrides so we can add analytics to docs website.

Reply:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#custom-site-analytics